botbot/apps/plugins/runner.py

```diff
@@ -187,11 +204,26 @@ def register(self, plugin):
                 continue
             if (not key.startswith('__') and
                     getattr(attr, 'route_rule', None)):
-                LOG.info('Route: %s.%s listens to %s for matches to %s',
-                         plugin.slug, key, attr.route_rule[0],
-                         attr.route_rule[1])
-                getattr(self, attr.route_rule[0] + '_router').setdefault(
-                    plugin.slug, []).append((attr.route_rule[1], attr, plugin))
+                if attr.route_rule[0] == "commands":
```

Comment:
since you are using ```attr.route_rule[0]``` and 1 a couple of times, it is better to assign it to a variable